lib/taskana-core/src/main/java/pro/taskana/common/internal/util/LogSanitizer.java

```diff
@@ -0,0 +1,13 @@
+package pro.taskana.common.internal.util;
+
+public class LogSanitizer {
+
+  private LogSanitizer() {
+    throw new IllegalStateException("Utility class");
+  }
+
+  // Remove characters which break the log file pattern. Protects against injection attacks.
+  public static String sanitizeLoggingMessage(String loggingMessage) {
```

Comment:
make the method generic, so that you don't have to call toString() from the outside?